helm/templates/monitoring.yaml

```
@@ -0,0 +1,211 @@
+spec:
+    replicas: 1
+    selector:
+        matchLabels:
+            app: prometheus
+    template:
+        metadata:
+            labels:
+                app: prometheus
+        spec:
+            containers:
+                -   name: prometheus
```

YAMLlint (1.37.1)

[warning] too many spaces after hyphen

(hyphens)